src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -239,4 +240,12 @@ export class CalciteAccordionItem {
       this.el
     );
   }
+
+  // private getIcon(): string {
```

Comment:
Do we need this commented-out code?

Reply:
Nope. Will remove the commented out part.